capstone-frontend/src/components/CreateList.jsx

```diff
@@ -64,22 +135,80 @@ class CreateList extends Component {
             <Form>
-              <Form.Group controlId="createBookListForm">
+              <Form.Group controlId="form-booklist-name">
                 <Form.Label>Name of Booklist</Form.Label>
                 <Form.Control type="text" placeholder="Enter Name" />
               </Form.Group>
-              <Button variant="primary" type="submit" onClick={() => this.handleSubmit()} disabled={this.state.loading}>
-                Create Booklist
-                    {this.state.loading &&
-                  <Spinner
-                    as="span"
-                    animation="border"
-                    size="sm"
-                    role="status"
-                    aria-hidden="true"
-                    className="ml-4"
-                  />}
-              </Button>
+              <Form.Group controlId="form-search-term">
+                <Form.Label>Search for Books</Form.Label>
+                <Form.Control type="text" placeholder="Search for books to add" onChange={(event) => this.handleSearchTermChange(event)} />
+                {this.state.fetchingBooks &&
+                  <div className="text-center">
+                    <Spinner
+                      as="span"
+                      animation="border"
+                      size="lg"
+                      role="status"
+                      aria-hidden="true"
+                      className="my-5"
+                    />
+                  </div>}
+              </Form.Group>
+
+              {
+                this.state.displayBooks &&
+
+                <div>
+                  <h3 clasName="my-4 px-4">Search Results</h3>
+                  <Row className="px-3 text-center">
+                    {this.state.searchResults.map(book =>
+                      <Col md={3} className="px-2 my-0 border" style={{ borderColor: "#ccc" }} key={book.id}>
+                        <img className="img-responsive mt-3 p-0 rounded" src={book.thumbnailLink} alt={book.title} />
+                        <h5 className="mt-4"> {book.title} </h5>
+                        <p className="my-1"> {book.authors.join(', ')} </p>
+                        {this.state.addedBooksIDs.includes(book.id) ?
+                          <Button className="my-5" variant="danger" onClick={() => this.removeBookFromList(book)}>Remove Book</Button>
+                          :
+                          <Button className="my-5" onClick={() => this.addBookToList(book)}>Add to Booklist</Button>}
+                      </Col>
+                    )}
+                  </Row>
+                </div>
+              }
+
+              {
+                (this.state.addedBooks.length !== 0) &&
+
+                <div>
+                  <h2 className="text-center my-4 px-4 ">Added Books</h2>
+                  <Row className="text-center px-3">
+                    {this.state.addedBooks.map(addedBook =>
+
+                      <Col md={3} className="px-2 my-0 border" style={{ borderColor: "#ccc" }} key={addedBook.id}>
```

Comment:
Is it possible to put this borderColor: "#ccc" in a CSS class? Usually direct style elements shouldn't go in the html

Reply:
ok